app/uk/gov/hmrc/agentclientrelationships/services/RelationshipsService.scala

```diff
@@ -179,19 +187,20 @@ class RelationshipsService @Inject()(gg: GovernmentGatewayProxyConnector,
     }
   }
 
-  def lookupGGForOldRelationship(arn: Arn, clientVrn: Vrn)(
+  def lookupESForOldRelationship(arn: Arn, clientVrn: Vrn)(
     implicit ec: ExecutionContext, hc: HeaderCarrier, request: Request[Any], auditData: AuditData): Future[Set[AgentCode]] = {
     auditData.set("vrn", clientVrn)
 
     for {
-      agentCodes <- gg.getAllocatedAgentCodesForHmceVatDec(clientVrn)
+      agentGroupIds <- es.getDelegatedGroupIdsFor(s"HMCE-VATDEC-ORG~VATRegNo~${clientVrn.value}")
```

Comment:
Knowlege of tax-enrolment's enrolment keys is leaking outside of the EnrolmentStoreProxyConnector here. 
Consider tweaking the method signature of getDelegatedGroupIdsFor(String) so it takes the component parts instead?